video/out/opengl/common.c

```diff
@@ -350,16 +360,19 @@ static const struct gl_functions gl_functions[] = {
     },
     {
         .ver_core = 310,
+        .ver_es_core = 300,
         .extension = "GL_ARB_uniform_buffer_object",
         .provides = MPGL_CAP_UBO,
     },
     {
         .ver_core = 430,
+        .ver_es_core = 310,
         .extension = "GL_ARB_shader_storage_buffer_object",
         .provides = MPGL_CAP_SSBO,
     },
     {
         .ver_core = 430,
+        .ver_es_core = 310,
```

Comment:
The ComputeWorkGroup attib vary across GL and GLES. At least `MaxComputeWorkGroupInvocations` on big GL is 1024, and GLES can be as low as 128.

Might want to doube-check that those are honoured through the mpv codebase.

With that, this patch looks great.

Reply:
In this case, I believe it's a problem with other vendors and not nvidia. Nvidia are very strict and hide the extension if the GL version is too low.